src/main/java/org/spburegistry/backend/controller/ProjectController.java

```diff
@@ -18,11 +22,15 @@ public class ProjectController {
     @Autowired
     private ProjectService projectService;
 
-    @GetMapping
-    public Iterable<ProjectTO> getAllProjects() {
-        return projectService.findAll();
+    @GetMapping("/projects")
```

Comment:
There is path /data/projects already in @RequestMapping and it will be added to all paths, so do not add /projects to your @GetMapping. Now your path looks like this: loacalhost:8080/data/projects/projects